packages/iov-cli/src/cli.ts

```diff
@@ -82,7 +82,7 @@ export function main(originalArgs: ReadonlyArray<string>): void {
 
   let init = `
     import leveldown = require('leveldown');
-    import levelup = require('levelup');
+    import levelup from "levelup";
```

Comment:
This should work... we use `import levelup, { LevelUp } from "levelup";` in the wallet and it works great.

Reply:
Jap, I could reproduce the type bug and this diff fixes it.

This is probably broken since we switched levelup types from the custom types file to @types/levelup